CONTRIBUTING.md

```diff
@@ -32,6 +32,16 @@ To manually format the code:
 tox -e format
 ```
 
+## Pre-commit plugin
+
+To forget about coding style and let pre-commit fix your flake8/isort/black issue.
+
+```
+pre-commit install
```

Comment:
Maybe with a note on where to fetch pre-commit?

Reply:
Sure